### ISSUE TITLE
Homepage carousel

### ISSUE THREAD

**Issue by Frijol** [edgi-govdata-archiving/EEW-Website#304]
Create a carousel for the homepage that accepts and displays images and captions so we can rotate through whatever projects or events we're currently highlighting

**Comment by ericnost** [comment]
Assigned myself to finish the maps I started

**Comment by ericnost** [comment]
What do we think of these for the carousel? I adjusted the color ramps to match the website .scss

![eew_2020_cds_CWA](https://user-images.githubusercontent.com/6888951/145119795-d515308b-11bf-4d71-a208-ba907091e5c5.png)
![eew_2020_states_CAA](https://user-images.githubusercontent.com/6888951/145120226-50fa0d64-a241-4066-86e3-6e752e55c038.png)

**Comment by Frijol** [comment]
I wonder if the orange would be easier to interpret, as more of a "bad" color?

**Comment by ericnost** [comment]
Good point! I tried to create a 4 colour scheme (I realize we don't need 5 as in the above maps, 4 is enough) using the .scss accent #e56d13 but I think my colour blindness is getting in the way. Is this better? I want to avoid using white as the lowest class because that implies 0 violations (even though some states did report 0 violations, that's....misleading)

<img width="1287" alt="Screen Shot 2021-12-08 at 1 35 24 PM" src="https://user-images.githubusercontent.com/6888951/145264561-8732503a-72df-49d0-892a-10310dab3366.png">

**Comment by Frijol** [comment]
Yes I think so!

**Comment by ericnost** [comment]
Ok, how about this? Just keep me up to date about which specific folder I can pull to and I'll add these.
![eew_2020_cds_CWA](https://user-images.githubusercontent.com/6888951/145273501-2f6ac167-8a32-4b9f-b2ac-15ac871a5f18.png)
![eew_2020_states_CAA](https://user-images.githubusercontent.com/6888951/145273502-3df96a12-7fe9-46ae-a4a1-7e7d116937b4.png)
